src/components/popup/popup.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function Popup(props) {
```

Comment:
Should the popup component contain the styling for popups? e.g. absolute positioning, borders, background, etc.